### ISSUE TITLE
get overlapping species (and clean the names)

### ISSUE THREAD

**Issue by lizzieinvancouver** [lizzieinvancouver/egret#31]
@wangxm-forest please get the overlapping species for: USDA x EGRET x Baskin x OSPREE (and relevant subsets, like OSPREE x EGRET, EGRET x USDA etc.) 

And! Please clean the species names in USDA, Baskin.  Thank you!

**Comment by lizzieinvancouver** [comment]
We want: 

- OSPREE x EGRET, EGRET x USDA (which is 8 spp)
- OSPREE x EGRET
- OSPREE x USDA
- EGRET x USDA (for fun!) 
- OSPREE x (EGRET + USDA)
- Baskin x (EGRET + USDA)
- Baskin x EGRET
- Baskin x USDA

**Comment by wangxm-forest** [comment]
> OSPREE x EGRET

19 spp
> OSPREE x USDA

60 spp
> EGRET x USDA (for fun!)

17 spp
> OSPREE x (EGRET + USDA)

71 spp
> Baskin x (EGRET + USDA)

381 spp
> Baskin x EGRET

198 spp
> Baskin x USDA

199 spp

**Comment by lizzieinvancouver** [comment]
@wangxm-forest It would be great to know: the dormancy class breakdown for:
- EGRET
- USDA
- OSPREE x (EGRET + USDA)

Maybe you could make one figure with dormancy class on the x and somehow show the numbers that way (either one plot with three colors or three plots methinks)?

**Comment by wangxm-forest** [comment]
@lizzieinvancouver 
![DormancyClass_Egret](https://github.com/lizzieinvancouver/egret/assets/81341625/5d8c3e93-3135-4c52-ad5d-3cb30c6d82ab)
![DormancyClass_OSPREExEgretUSDA](https://github.com/lizzieinvancouver/egret/assets/81341625/97a82990-52ad-4c64-940f-3a56c5ab9510)
![DormancyClass_USDA](https://github.com/lizzieinvancouver/egret/assets/81341625/eecbf552-ba59-4b71-991e-398157d4ef86)

**Comment by lizzieinvancouver** [comment]
@wangxm-forest This would be great to update early in the retreat when everything seems cleaned.

**Comment by wangxm-forest** [comment]
@lizzieinvancouver Just realized that we might want to update the OSPREE species name with WorldFlora before we get overlapping species?

**Comment by lizzieinvancouver** [comment]
@wangxm-forest Good idea! Yes, we should. I think @dbuona had already started pulling over stuff we would need from OSPREE so see if you can find where that is ... and should discuss where to put OSPREE stuff in EGRET.

**Comment by wangxm-forest** [comment]
Updated overlapping sp:
OSPREE x EGRET: 8
OSPREE x USDA: 41
EGRET x USDA: 16
Baskin x EGRET: 207
Baskin x USDA: 172
OSPREE x (EGRET + USDA): 48

**Comment by wangxm-forest** [comment]
![Image](https://github.com/user-attachments/assets/99268c24-c934-4689-9ba9-c78c73e4cd14)
![Image](https://github.com/user-attachments/assets/fd415d60-e5d4-487f-83e2-e541b4a62fa2)
![Image](https://github.com/user-attachments/assets/53817d72-1cd8-4e51-8226-6dd28f9ff1b5)
![Image](https://github.com/user-attachments/assets/37c19556-e9d7-4322-95de-3e0a0d8092ac)

![Image](https://github.com/user-attachments/assets/83485276-1a2f-4e8f-a6f0-8ae4ec28acb5)
![Image](https://github.com/user-attachments/assets/69d9c623-d357-4fe2-8b9e-37903327008b)
![Image](https://github.com/user-attachments/assets/bc303953-0ec0-41ad-8f94-5142ddd5ac49)

**Comment by vvandermeersch** [comment]
@wangxm-forest  In `baskinegretclean.csv`, _Sinapis arvensis_ still has two rows (the correct genus is Sinapis):

| Original.Genus | Corrected.Genus | Genus_species    | Family       | Dormancy.Class | species          |
|----------------|-----------------|------------------|--------------|----------------|------------------|
| Brassica       | Brassica        | Sinapis arvensis | Brassicaceae | PD             | Sinapis_arvensis |
| Sinapis        | Sinapis         | Sinapis arvensis | Brassicaceae | PD             | Sinapis_arvensis |

**Comment by wangxm-forest** [comment]
@vvandermeersch I see. I used the unique() function before correcting the species name, but it seems that these two species were changed to have the same name. I will run the unique() function again at the end. Thank you for pointing that out!

**Comment by wangxm-forest** [comment]
Updated overlapping sp:
OSPREE x EGRET: 9
OSPREE x USDA: 66
EGRET x USDA: 26
Baskin x EGRET: 207
Baskin x USDA: 173
OSPREE x (EGRET + USDA): 69

**Comment by wangxm-forest** [comment]
<img width="533" height="430" alt="Image" src="https://github.com/user-attachments/assets/357c33fb-fefa-4645-8af7-25f68d67a083" />

<img width="533" height="430" alt="Image" src="https://github.com/user-attachments/assets/4e8df53f-cbec-4bd6-b9b9-dffd41ab7eb0" />

<img width="533" height="430" alt="Image" src="https://github.com/user-attachments/assets/138f84ac-db79-48df-870f-a6f7a87f20c3" />

<img width="533" height="430" alt="Image" src="https://github.com/user-attachments/assets/576d7c69-2bac-48e0-8f37-2fb07c80b068" />

<img width="533" height="430" alt="Image" src="https://github.com/user-attachments/assets/d7128355-db85-4e85-9f1a-b36550e18de4" />

<img width="533" height="430" alt="Image" src="https://github.com/user-attachments/assets/ecf914cb-8109-4a88-a49d-378f3256c5c2" />

<img width="533" height="430" alt="Image" src="https://github.com/user-attachments/assets/67be9651-94ad-4406-836e-e7a7f6a49a86" />
